articles/openshift/oshift-trials-gs.md

```diff
@@ -0,0 +1,67 @@
+
+During a trial each cluster is also provisioned with 2 small run time packs. These are essentially 2 OpenStack VM's running RedHat atomic OS with 16GB RAM, 2vCPU's and 100GB disks each. This is also the minimum number of Runtime Packs UKCloud recommends in each OpenShift deployment in order to provide minimal resilience and enable interruption free patching. After the trial phase it is possible to deploy larger and more nodes for your cluster.
+
+**OpenShift documentation:** [*Architecture*](https://docs.openshift.com/container-platform/3.11/architecture/index.html)
+
+## GUI and CLI Access
+
+In order to access the OpenShift cluster GUI customers require a internet-enabled device running a HTML5 capable browser and need to be set up for access in the UKCloud Portal. Customers will also be set up with the relevant access in their OpenShift deployment.
+
+**OpenShift documentation:** [*Web Console Walkthrough*](https://docs.openshift.com/container-platform/3.11/getting_started/developers_console.html)
+
+In addition to GUI access customers can also access OpenShift through the CLI, in order to do this customers need to have installed the OC client on their machines. 
+
+**OpenShift documentation:** [*Installing the OC CLI*](https://docs.openshift.com/container-platform/3.11/cli_reference/get_started_cli.html#installing-the-cli)
+
+## Storage
+
+Each trial deployment has a block storage quota of 2TB across either tier 1 or tier 2 storage. This is enforced at the OpenStack project layer. The storage is available to the projects customers create within OpenShift, customers can consume that storage by creating Persistent Volume Claims (PVCs) for their applications. It is good practice to use PVCs for any persistent storage requirements as any data stored within local volumes will be lost when pods are moved from one node to another; for example as a result of node evacuations during patching. Storage quotas can be increased by creating a service request in the UKCloud portal.
+
+OpenShift is deployed with an internal image registry making use of UKCloud's object storage service for persistence.  This includes 60GB of free storage capacity which does not come out of a customer's 2TB block storage capacity, rather this is in addition to the 2TB data limit. UKCloud do provision ElasticSearch, Fluentd and Kibana stack which uses 20GB, Hawkular Metrics backed by a Cassandra database which uses 10GB and the Prometheus Operator which uses 106GB of tier 2 storage (which consumes some of the 2TB storage limit).
+
+## Image Security
+
+Customers should take care when creating their container images to ensure the images do not run as root. Containers should be set up to run as a user which has just enough rights to run the container workload.
```

Comment:
You should take care when creating your container images to ensure the images do not run as `root`. You should set up containers to run as a user who has just enough rights to run the container workload.